src/Components/Card/style.scss

```
@@ -0,0 +1,22 @@
+.cards-list_row {
+  width: 100%;
+  padding: .3rem 1rem .3rem 1rem;
```

```suggestion
  padding: 0.3rem 1rem 0.3rem 1rem;
```
дурной тон экономить на нулях. но здесь религиозная война

---

не будем воевать